extensions/sql-migration/src/dialog/skuRecommendationResults/skuRecommendationResultsDialog.ts

```diff
@@ -26,30 +26,13 @@ export class SkuRecommendationResultsDialog {
 	// Dialog Name for Telemetry
 	public dialogName: string | undefined;
 	private _disposables: vscode.Disposable[] = [];
-	public title: string;
-	public targetName: string;
+	public title?: string;
+	public targetName?: string;
 
-	public targetRecommendations: mssql.SkuRecommendationResultItem[];
+	public targetRecommendations?: mssql.SkuRecommendationResultItem[];
 
-	constructor(public model: MigrationStateModel, public _targetType: MigrationTargetType, public recommendations: mssql.SkuRecommendationResult) {
-		switch (this._targetType) {
-			case MigrationTargetType.SQLMI:
-				this.targetName = constants.AZURE_SQL_DATABASE_MANAGED_INSTANCE;
-				this.targetRecommendations = recommendations.sqlMiRecommendationResults;
-				break;
-
-			case MigrationTargetType.SQLVM:
-				this.targetName = constants.AZURE_SQL_DATABASE_VIRTUAL_MACHINE; // AZURE_SQL_DATABASE_VIRTUAL_MACHINE_SHORT
-				this.targetRecommendations = recommendations.sqlVmRecommendationResults;
-				break;
-
-			case MigrationTargetType.SQLDB:
-				this.targetName = constants.AZURE_SQL_DATABASE;
-				this.targetRecommendations = recommendations.sqlDbRecommendationResults;
-				break;
-		}
+	constructor(public model: MigrationStateModel, public _targetType: MigrationTargetType) {
```

Comment:
So I took `recommendations` out of the constructor, but left `_targetType` yet it still seems to work properly. Not sure how that works honestly but if you have any suggestions that'd be great, but if not we could probably leave it as is 🤷‍♂️

Reply:
we can keep this since it works. I think it might be cleaner if we keep that switch statement to set the targetName and title values in the constructor. `targetRecommendations` can be set in openDIalog

Reply:
done!